czo2hs.py

```diff
@@ -157,17 +158,17 @@ def migrate_czo_row(czo_row_dict, row_no=1, progress_report=5):
                                                                        error_item["error_msg_list"]])))
     logging.info(text_emphasis("CZO_ID <---> HS_ID Lookup Table"))
     logging.info(czo_hs_id_lookup_df.to_string())
-    results_file = os.path.join(logdir, 'czo_hs_id_{}.csv'.format(start_time))
-    logging.info("*" * 20 + "Saving Lookup Table to {}".format(results_file) + "*" * 20)
+    czo_hs_id_lookup_csv_path = os.path.join(logdir, 'czo_hs_id_{}.csv'.format(start_time_str))
+    logging.info(text_emphasis("Saving Lookup Table to {}".format(czo_hs_id_lookup_csv_path)))
 
-    czo_hs_id_lookup_df.to_csv(results_file, encoding='utf-8', index=False)
+    czo_hs_id_lookup_df.to_csv(czo_hs_id_lookup_csv_path, encoding='utf-8', index=False)
 
```

Comment:
shorter, friendlier names, please. czo hs csv are kind of known. What's specifically unique here?

Reply:
rename czo_hs_id_lookup_csv_path to results_file it's more readable

Reply:
fixed in new commit